api-specs/test/api.raml

```diff
@@ -475,6 +475,21 @@ uses:
                 type: object
                 example: !include examples/action-success.json
 
+    /json-serializable-primitive-type:
+      put:
+        description: Use the PUT method to write data to a backend system. Any JSON serializable payload is accepted. The following request example adds a product to a cart. For the response, we recommend to use standard HTTP codes and `application/json` encoded content. The response will be structured [as defined by the `body` property of the action](/). The following response example contains the updated cart information, which includes the added product.
+        body:
+          application/json:
+            type: string
+            example: !include examples/action-payload-1.json
+        responses:
+          200:
+            description: We recommend to use standard HTTP response codes and `application/json` encoded content. The response will look like the response you have declared in your action. As an example we will fetch the cart.
+            body:
+              application/json:
+                type: object
```

Comment:
The response type of the API described as "any JSON serializable payload"  would be `any`, not `object`.     E.g. `"just a string"` is valid JSON, or  `[1, 2, "string", false]` is, too.